src/render/pass/SurfacePassElement.hpp

```diff
@@ -56,6 +56,7 @@ class CSurfacePassElement : public IPassElement {
     virtual bool                needsPrecomputeBlur();
     virtual std::optional<CBox> boundingBox();
     virtual CRegion             opaqueRegion();
+    virtual CRegion             visibleRegion(bool& cancel);
```

Comment:
why is this virtual? What is it overriding?

Reply:
whoops